### DISCUSSION TITLE
API Support

### DISCUSSION THREAD

**Discussion by oguzhantortop** [majusko/pulsar-java-spring-boot-starter#292] [Ideas]
Do we have any API support currently? Spring style readiness and health check APIs maybe? Also taking a list of running consumers and ability of stopping and starting them by API would be nice?

**Comment by majusko** [comment]
There is support for Prometheus endpoints in the next 1.2.0 release and API from Spring Boot Actuator and Micronaut.

![185542409-78d39c74-f89d-4a52-bc9c-c35a43718574](https://user-images.githubusercontent.com/2635406/210905514-b0cb7c19-5551-4abb-8985-24f8c9e99b9c.png)

For any other information, I recommend using this: https://pulsar.apache.org/docs/2.10.x/administration-pulsar-manager/

**Reply by oguzhantortop** [reply]
Actually, I meant an API to stop and start consumers from API. I think this might be useful. Pulsar Admin APIs don't cover such things... Also for shared subscriptions, we might increase the number of consumers...

**Reply by majusko** [reply]
I see, ok that makes sense. I can do some research on how to do something like that.

**Reply by oguzhantortop** [reply]
I can also have a look at it...
